extension/src/background/helpers/dataStorage.ts

```diff
@@ -24,15 +24,19 @@ const storage = browser.storage as BrowserStorage;
 // browser storage uses local storage which stores values on disk and persists data across sessions
 // session storage uses session storage which stores data in memory and clears data after every "session"
 // only use session storage for secrets or sensitive values
-export const localStorage = storage?.local;
```

Comment:
we need the name `localStorage` available so we can call `window.localStorage` later

Reply:
ah yup 😄